### DISCUSSION TITLE
Project

### DISCUSSION THREAD

**Discussion by Alexandra21bi2** [nonatalies/WeLoveJavaFX#39] [General]
Доброй ночи! Предлагаю вести всё обсуждение проекта здесь. Мне кажется такой формат будет удобным.

**Comment by nonatalies** [comment]
Доброй ночи, Александра! Полностью поддерживаю Вашу инициативу!

**Reply by Alexandra21bi2** [reply]
Наталия, спасибо за проявленную активность и поддержку!

**Reply by nonatalies** [reply]
Все для проекта! #WeLoveJavaFX

**Comment by annikiforovaa** [comment]
Доброй ночи! Поддерживаю :)

**Reply by Alexandra21bi2** [reply]
Анна, отлично! Спасибо за ответ!

**Reply by Orekhvera** [reply]
Добрый день)

**Comment by Orekhvera** [comment]
Как наши успехи?)))))))))))))

**Reply by Alexandra21bi2** [reply]
Вера, молодец, что загрузила архитектуру.

**Reply by nonatalies** [reply]
Александра, подтвердите, что Commited с одной t - это наша командная фишка

**Reply by Alexandra21bi2** [reply]
Подтверждаю! Подпись прилагается в беседе в социальной сети Вконтакте.

**Comment by Alexandra21bi2** [comment]
Нам необходима ручная документация: список участников, роли, основные фичи, цель проекта, описание.
Сдача финальная 7 июня. Документацию я сделаю
